test/plugins/removeUselessDefs.05.svg

```diff
@@ -0,0 +1,23 @@
+Don't remove used symbols.
```

Comment:
Could you please explain this test?

It doesn't look like it would've been broken before, and it doesn't feature a `defs` node or other hidden element.

Just want to check was this added due to a known issue, or were you just playing it safe and checking more cases?

Reply:
Basically playing it safe. Since `<g>` is neither in the non-rendering set nor a `<def>`, just wanted to make sure nothing weird happened in that case.